docs/source/examples/webserver.py

```diff
@@ -0,0 +1,74 @@
+#!/usr.bin/python3
+import logging
+import os
+
+from conu.backend.docker.container import DockerRunCommand
+
+from conu import DockerBackend
+from conu.utils import random_str
+from conu.utils.filesystem import Directory
+
+
+# this is the port where we can access our app
+port = 8765
+
+
+# convenience function to run our application - webserver
+def run_container(local_dir):
+    """
+    serve path `local_dir` using the python http webserver in a docker container
+    :param local_dir: str, path to the directory, it should exist
+    :return: instance of DockerContainer
+    """
+    image_name = "registry.fedoraproject.org/fedora"
+    image_tag = "27"
+
+    # we'll run our container using docker engine
+    backend = DockerBackend(logging_level=logging.DEBUG)
+    image = backend.ImageClass(image_name, tag=image_tag)
+
+    # is the image present?
+    try:
+        image.get_metadata()
+    except Exception:
+        image.pull()
```

Comment:
This might deserve own method in the future. Sth like image.is_pulled/present/in_local_registry. And maybe method image.is_up_to_date.

Reply:
I agree

Reply:
https://github.com/fedora-modularity/conu/issues/82